content/docs/accessibility.md

```diff
@@ -306,7 +306,7 @@ as the `window` object never receives a `click` event. This can lead to obscured
 
 <img src="../images/docs/outerclick-with-keyboard.gif" alt="A toggle button opening a popover list implemented with the click outside pattern and operated with the keyboard showing the popover not being closed on blur and it obscuring other screen elements." />
 
-The same functionality can be achieved by using an appropriate event handlers instead, such as `onBlur` and `onFocus`:
+The same functionality can be achieved by using an appropriate event handler instead, such as `onBlur` and `onFocus`:
```

Comment:
Yeah this is wrong. In the example below this, both `onFocus` and `onBlur` have been implemented, so perhaps `using appropriate event handlers` will be better.